go/chat/signencrypt/codec.go

```diff
@@ -450,17 +454,17 @@ func (r *codecReadWrapper) Read(callerBuf []byte) (int, error) {
 	return 0, io.EOF
 }
 
-func NewEncodingReader(encKey SecretboxKey, signKey SignKey, nonce Nonce, innerReader io.Reader) io.Reader {
+func NewEncodingReader(encKey SecretboxKey, signKey SignKey, signaturePrefix libkb.SignaturePrefix, nonce Nonce, innerReader io.Reader) io.Reader {
```

Comment:
Let's throw in a doc comment explaining what the prefix is for. Like: "The prefix should usually be something like the name of your application. The sender and recipient must use the same prefix for a signature to be valid. This prevents an attacker from replaying signatures from one application in another application, when the two applications are using shared keys but have different rules about what a signature means. A prefix must not contain the null character."